src/components/Pricing/index.tsx

```diff
@@ -3,6 +3,25 @@ import TableDesktop from './Table/TableDesktop';
 import { PricingInfo } from '../../content/pricing/config';
 import PricingCard from '@components/PricingCard';
 import { Text } from '@components/LandingPage/Text';
+import Accordion from '@components/Accordion';
+import settings from '@base/settings.json';
+import data from './data.json';
+
+function replacePlaceholders(
+  content: string,
+  resources: { [key: string]: string },
+): string {
+  return content.replace(/\$([a-zA-Z0-9]+)/g, (match, key) => {
+    return resources[key] || match;
+  });
+}
+
+const resources = settings.support.resources || {};
+
+const processedFaqs = data.faqs.map((faq) => ({
+  ...faq,
+  content: replacePlaceholders(faq.content, resources),
+}));
```

Comment:
If you're trying to replace just the content key value

```suggestion
const processedFaqs = data.faqs.map(({label, content}) => ({
  label,
  content: replacePlaceholders(content, resources),
}));
```